components/ProfileModal/Profile-Modal.tsx

```
@@ -0,0 +1,298 @@
+              Rating: {influencer?.backend?.rating}
+            </Text>
+          </View>
+        </View>
+      </View>
+      <View style={styles.chipContainer}>
+        {influencer?.profile?.category &&
+          influencer?.profile?.category.map((interest, index) => (
+            <Chip key={index} style={styles.chip} mode="outlined">
+              {interest}
+            </Chip>
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)